brayns/common/scene/Model.h

```diff
@@ -272,6 +292,18 @@ class Model
     Boxf _bounds;
     bool _useSimulationModel{false};
 
+    struct SDFGeometryData
+    {
+        std::vector<SDFGeometry> geometries;
+        std::map<size_t, std::vector<uint32_t>> geometryIndices;
+
+        std::vector<std::vector<size_t>> neighbours;
+        std::vector<size_t> neighboursFlat;
+        bool isDirty{false};
```

Comment:
the dirty flag for other types of data is kept separately, maybe do the same for consistency..?

Reply:
Yes, fixed.